src/textual/widgets/_tabs.py

```diff
@@ -107,7 +107,7 @@ class Tab(Static):
     }
     Tab:disabled {
         color: $text-disabled;
-        text-opacity: 50%;
+        text-opacity: 70%;
```

Comment:
This accounts for the now-correct calculation of the final text color when opacity is present.